Gemfile

```diff
@@ -71,3 +71,11 @@ group :development do
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem "spring"
 end
+
+group :test do
+  gem 'capybara'
```

Comment:
Capybara is a gem we are going to need, but not yet. Let's Keep it Simple by not adding gems we aren't going to use right away. Later in the project we have a group project to setup capybara so kindly remove this gem and do: bundle install